### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by Smitholi67**
Added new link for how to contribute to CNTi.

### CODE REVIEW

**Review by rich-l — APPROVED** [review]
lgtm

**Review by taylor — APPROVED** [review]
Lgtm

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The [CNTi Charter](https://github.com/lfn-cnti/cnti/blob/c93c0172376ac5ccb1e50f3
 
 ## Get Involved
 
-Learn more on how to get involved by visiting the [CNTi Best Practices Wiki](https://wiki.lfnetworking.org/x/HAAxBw)
+Learn more on how to get involved by visiting the [CNTi Best Practices Wiki](https://wiki.lfnetworking.org/x/HAAxBw) and reading about ways to [contribute](https://github.com/lfn-cnti/cnti/blob/46c6ced28abc378c3b97d95e59661957ab5b80c0/CONTRIBUTING.md) to CNTi.
```
Comment (martin-mat) [inline README.md]: https://github.com/lfn-cnti/cnti/blob/main/CONTRIBUTING.md

**Review by martin-mat — CHANGES_REQUESTED** [review]

**Review by martin-mat — APPROVED** [review]
lgtm, thanks.

**Pull request merged**